py/torch_tensorrt/dynamo/conversion/impl/normalization/ops.py

```diff
@@ -47,8 +47,20 @@ def batch_norm(
     if has_dynamic_shape(input.shape):
         assert input.shape[1] != -1, "Channel dim can't be dynamic for batch norm."
 
+    if weight is None:
+        weight = np.array(1.0)
+
+    if bias is None:
+        bias = np.array(0.0)
+
+    if running_mean is None:
+        running_mean = np.array(0.0)
+
+    if running_var is None:
+        running_var = np.array(1.0)
```

Comment:
For these, it should be okay to not cast to `np.array` in the converter (instead leave them as ints or floats), since `to_numpy` should dictate this casting behavior for ints and floats. Specifically, one small difference is that I think `np.array(1.0)` has shape `()` (0D), but `to_numpy` generally adds a dimension, to make it 1D.